download.go

```diff
@@ -219,3 +238,37 @@ func downloadFile(authStr, url, filePath string) error {
 
 	return nil
 }
+
+func downloadBlob(authStr, url, filePath string) error {
+	log.Println("Downloading", url)
+
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", url, nil)
+
+	if err != nil {
+		return err
+	}
+
+	req.Header.Add("Authorization", authStr)
+	req.Header.Add("User-Agent", "go-gl/glow")
+	resp, err := client.Do(req)
+
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	var blob blobContent
```

Comment:
downloadBlob & downloadFile are particularly similar, perhaps we could have a single implementation? fileContent & blobContent could both implement Content()?